lib/client/product.rb

```diff
@@ -37,10 +37,10 @@ def update_batch(payload, params={})
                   e.response[:status],
                   'update product',
                   'Failed to update product batch',
+                  payload.map { |p|
+                    { sku: p[:sku] }
+                  },
                   {
-                    product_identifiers: payload.map { |p|
-                      { id: p[:id], sku: p[:sku] }
-                    },
```

Comment:
@Jhalter5Stones Why are we removing `id` here?

Reply:
@mrogers-5s id was null for some reason. Maybe the name is different at this point?

Reply:
The ID _may_ be `null` for new products, but it shouldn't be `null` for existing products.

Reply:
@mrogers-5s I think I fixed this. Let me know if it looks right now.